taskcluster/kinds/complete/kind.yml

```
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - build
+
+transforms:
+    - ffios_taskgraph.transforms.bitrise
+    - taskgraph.transforms.task
+
+task-defaults:
+    worker-type: bitrise
```

Bug: In order to fix 

```
scriptworker_client.exceptions.TaskVerificationError: No workflow scopes starting with 'project:mobile:firefox-ios:releng:bitrise:' found
```

the worker-type should be something else than bitrise. We don't need to run the complete task on the Bitrise infra given that this task just checks some Taskcluster metadata

```suggestion
    worker-type: misc
```